extra_docs/realtime/channels.md

```diff
@@ -0,0 +1,131 @@
+## Communication with channels
+
+Client-side applications can communicate with each other in real-time using channels which basically, allows sending and receiving messages through websocket. 
+Before using this feature, you need to create at least one channel and an appropriate policy with allowed "publish" and "subscribe" actions. Channels can be managed with [Jexia Web Management Application](https://docs.jexia.com/getting-started/user-management/).  
+
+When an application sends a message to the channel, everyone subscribed to that channel immediately receives it. If the persistent store is activated for the channel (using management application), it is also possible to get messaging history.   
+  
+### Initialize  
+To use real-time communication initialize Jexia client with the real-time module:  
+  
+``` javascript
+import { jexiaClient, realTime } from "jexia-sdk-js";  
+  
+const rtm = realTime();  
+  
+jexiaClient().init({  
+  projectID: "your-project-id",  
+  key: "your-api-key",    
+  secret: "your-api-secret"  
+}, rtm);  
+```  
+
+Then you can get your channel object (considered that channel has been already created in management application, otherwise an error will be thrown, see "Handling errors" below):
+
+```javascript
+const channel = rtm.channel("my_channel");
+```
+
+  
+### Subscribe to the channel events
+Basically, channel object is inherited from the [RxJS Observable](https://rxjs.dev/api/index/class/Observable), so just use `subscribe()` method: 
+  
+```javascript
+channel.subscribe(message => {
+  // you received a message here!
+});
+```
+
+You can use `.pipe()` and any RxJS operators to have more flexibility:
+
+```javascript
+channel.pipe(
+  filter(message => message.from === user.id), // filter only specific user
+  pluck("data") // pull out only data field 
+).subscribe(data => {
+  // you've got data from the specific user here!
+});
+```  
+ 
+ ### Unsubscribe
+
+ Don't forget to unsubscribe in case there is no needs to get messages from the channel anymore, just the use default RxJS `unsubscribe()` method:
+ 
+ ```javascript
+ const subscription = channel.subscribe();
+subscription.unsubscribe();
+ ``` 
+  
+### Publish to the channel
+
+You can publish any valid JSON type to the channel. Here is publishing a string example:
+  
+``` javascript
+const message = "Hey bro, did you feed our home rhino?";  
+channel.publish(message); 
+```  
+
+It is also can be an object:
+```javascript
+channel.publish({
+  product: "apple",
+  amount: 42
+});  
+```
+  
+### Get the message history  
+  
+```javascript
+channel.getLog().subscribe(log => {
+  // you will receive an array of all the messages here (see schema below)
+});  
+```
+
+It is also possible to use filters:
```

Comment:
Can we add link to filter description in JS SDK?  To provide understanding to client what is possible?

Reply:
No, we are using `compodoc` tool and it doesn't allow to create a cross links

Reply:
where exactly should this link point to? I believe we could make some relative links 🤔

Reply:
Well, since we need to create a link the doc **itself**, I don't think it's going to work. We might create a static link to  github.io (where the doc is actually hosted), but then any change will break this link.